[hosting-policy] Make BCR required (#85)
bazel-contrib/SIG-rules-authors: hosting-policy.md

1. Must publish semver releases.
    - Optional: follow the same release pattern as the rules-template does.
1. Must work with LTS Bazel version

> 1. Include the rules in bazel-central-registry, keep that CI green

Reviewer: ```suggestion
1. Must publish the rules to the [Bazel Central Registry](https://github.com/bazelbuild/bazel-central-registry), keep that CI green
```
